src/h5web/providers/models.ts

```diff
@@ -158,4 +156,5 @@ export type Value<D extends Dataset> = D['shape'] extends ScalarShape
   ? PrimitiveType<D['type']>[]
   : never;
 
-export type ComplexArray = (ComplexArray | Complex)[];
+export type H5WebComplex = [number, number];
+export type ComplexArray = (ComplexArray | H5WebComplex)[];
```

Comment:
The new `Complex` type is called `H5WebComplex` temporarily. Once `complex.js` is removed, I intend to rename it `Complex`